### PR TITLE
chore: Use class syntax in `src`

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

**Comment by DominicKramer** [comment]
PTAL

File: test/fixtures/configuration.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-var Configuration = require('../../src/configuration.js');
+var Configuration = require('../../src/configuration.js').Configuration;
```
Comment (alexander-fenster) [inline test/fixtures/configuration.js]: was it renamed to configuration.ts?

Comment (alexander-fenster) [reply test/fixtures/configuration.js]: same below with error-message.js and others. please verify the paths are correct

Comment (DominicKramer) [reply test/fixtures/configuration.js]: The file was renamed to `configuration.ts`, but the test code references the transpiled js version of the file.

Comment (DominicKramer) [reply test/fixtures/configuration.js]: The same is true for `error-message.js`.

Comment (alexander-fenster) [reply test/fixtures/configuration.js]: I see. Makes sense.

**Pull request merged**